integration-test/Integration.Tests.Android.Adb.ps1

```diff
@@ -0,0 +1,418 @@
+# Integration tests for Sentry Unreal SDK on Android
```

Comment:
I assume this would be moved to app-runner as a new provider, in which case both android integration test files would be merged to a single one, right?

Reply:
Yes, tests cases are identical for both Adb and SauceLabs so there's no need to keep separate Tests.ps1 scripts for them. It will probably also make sense to introduce a parameter allowing to specify which provider to use.